src/constants.js

```diff
@@ -2,6 +2,7 @@
 
 const path = require('path');
 const userHome = require('./util/user-home-dir').default;
+const isWebpackBundle = require('is-webpack-bundle');
```

Comment:
I'm not sure we should use an external module for this - inlining it would allow webpack to remove the unneeded code at compile time.